File: starter.js
import fs from 'fs';

const processInputs = (path="inputs.txt") => {
  return fs
    .readFileSync("inputs.txt", "utf8")
    .split("\n")
    // .map(item => parseInt(item, 10));
    // .map(item => item.trim());
};

const part1 = (path) => {
  const inputs = processInputs(path);

};

const part2 = (path) => {
  const inputs = processInputs(path);

};

console.log(part1());
console.log(part2());

export { part1, part2 };

File: starter.py
import os


def parse_input(input: str):
    return input.strip()


def get_inputs():
    script_dir = os.path.dirname(os.path.realpath(__file__))
    file = open(f'{script_dir}/inputs.txt')
    inputs = [parse_input(line) for line in file.readlines()]
    file.close()
    return inputs


def part_1():
    inputs = get_inputs()

    return None


def part_2():
    inputs = get_inputs()

    return None


if __name__ == '__main__':
    print('Part 1:', part_1())
    print('Part 2:', part_2())
